demos/smart_classroom_demo_gapi/include/stream_source.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <opencv2/videoio.hpp>
+#include <opencv2/gapi/garg.hpp>
+
+namespace cv {
+namespace gapi {
+namespace wip {
```

Comment:
This is not a `cv::gapi` thing so no need to put these fake namespaces here. It is a custom input source written to handle the app specifics.